java/coverage/pom.xml

```diff
@@ -30,7 +30,7 @@
     <groupId>dev.feast</groupId>
     <artifactId>feast-parent</artifactId>
     <version>${revision}</version>
-    <relativePath>../..</relativePath>
+    <relativePath>..</relativePath>
```

Comment:
What does this module do exactly?

Reply:
It generates code coverage reports (JaCoCo) which presumably can be used in CI and in IDEs